src/ui/patch_hub/patchset_details_and_actions.sh

```diff
@@ -186,4 +190,7 @@ function handle_remove_bookmark_action()
   if [[ "$?" != 0 ]]; then
     create_message_box 'Error' 'Could not unbookmark patchset'$'\n'"- ${_patchset['patchset_title']}"
   fi
+
+  message_box='Deleted patchset:'$'\n'"- ${_patchset['patchset_title']}"$'\n'$'\n'
```

Comment:
Same nitpick, maybe say 'Removed bookmark from patchset: (...)' (or something better, but stress that it is being unbookmarked).